course_viewer/course_viewer/models.py

```diff
@@ -0,0 +1,111 @@
+        verbose_name = 'state'
+        verbose_name_plural = 'states'
+        db_table = 'states'
+
+
+class College(models.Model):
+    '''
+    The Transfer College
+    '''
+    college_code = models.CharField(
+                    max_length=15,
+                    primary_key=True,
+                    help_text='The code of the transfer college')
+    college_name = models.CharField(
+                        max_length=50,
+                        help_text='The name of the transfer college')
+
+    def __str__(self):
+        return self.college_name
+
+    class Meta:
+        ordering = ['college_code']
+        verbose_name = 'college'
+        verbose_name_plural = 'colleges'
+        db_table = 'colleges'
+
+
+class Course(models.Model):
+    '''
+    The Transfer Course
+    '''
+    transfer_course_id = models.CharField(
+                        max_length=15,
+                        help_text='The id of the transfer course')
+    transfer_course_credit = models.IntegerField(
+                    help_text='The number of credits the transfer course is')
+    transfer_course_number = models.CharField(
+                        max_length=15,
+                        help_text='The course number of the transfer course')
+    mtu_equiv_id = models.ForeignKey(
+                'MTUCourse',
+                related_name='mtu_equiv_id',
+                on_delete=models.CASCADE,
+                help_text='The MTU course associated with the transfer course')
+    transfer_course_college_code = models.ForeignKey(
+                            'College',
+                            related_name='transfer_course_college_code',
+                            on_delete=models.CASCADE,
+                            help_text='The college that the course is at')
+    transfer_course_state_code = models.ForeignKey(
+                            'State',
+                            related_name='transfer_course_state_code',
+                            on_delete=models.CASCADE,
+                            help_text='The state the course is located in')
+
+    def __str__(self):
+        return self.transfer_course_id
+
+    class Meta:
+        ordering = ['transfer_course_id']
+        verbose_name = 'transfer_course'
+        verbose_name_plural = 'transfer_courses'
+        db_table = 'transfer_courses'
+
+
+class MTUCourse(models.Model):
+    '''
+    The MTU Course
+    '''
+    mtu_course_id = models.CharField(
+                            max_length=50,
+                            help_text='The id of the MTU course')
+    mtu_course_name = models.CharField(
+                            max_length=50,
+                            help_text='The Name of the MTU course')
+    mtu_subject = models.CharField(
+                        max_length=50,
+                        help_text='The subject of the MTU course')
+    mtu_credits = models.IntegerField(
+                        help_text='The amount of credits the MTU course is')
```

Comment:
Some of the entries for `mtu_credits` earned are half credits.  Possibly use floats, then round on display for the integers?

Reply:
Same as last comment, blank=True and float field

Reply:
I think rounding would actually harm more than be helpful here. Seeing as how a 3 credit course at X college transferring as a .5 credit but displaying as 1 credit would be misleading in planning things like graduation credit fulfillment.